.travis.yml

```diff
@@ -58,6 +68,11 @@ after_success:
             docker push welder/web-e2e-tests
         fi
 
+        if [ "$COMMAND" == "cockpit-test" ] && [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
```

Comment:
```
if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
    docker login ...

    if [ "$COMMAND" == "end-to-end-test" ] ; then
       ... tag & push e2e related images
    fi

    if [ "$COMMAND" == "cockpit-test" ] ; then
       ... tag & push cockpit related images
    fi
```

Also note: docker push welder/web-with-coverage is not needed b/c the coverage image shouldn't be cached. This has been a mistake on my part.